Mirantis/virtlet
# Fix testability of libvirttools & improve build/cmd.sh

- Switch to Go 1.8.3
- Add proper abstraction for libvirttools - libvirt interaction
- Add "golden master" testing for libvirttools
- Use rsync in build/cmd.sh
- Fix flexvolume tests that were broken
- Run flexvolume tests during `make check`


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/mirantis/virtlet/297)
